pkg/openSearch/openSearchClient/error.go

```diff
@@ -50,16 +44,6 @@ func (o *OpenSearchResourceAlreadyExists) Error() string {
 	return o.Message
 }
 
-func NewOpenSearchResourceAlreadyExists(message string) *OpenSearchResourceAlreadyExists {
-	return &OpenSearchResourceAlreadyExists{
-		Message: message,
-	}
-}
-
```

Comment:
Why has this been removed?

Reply:
Fixed in c52072a0a084102ada43709c85b442b29bcf5ab2